### PR TITLE
Feature add service external ips

### PR DESCRIPTION
**Pull request by fusida**
Ⅰ. Describe what this PR does
Add service external ips
Ⅱ. Does this pull request fix one issue?

Ⅲ. List the added test cases (unit test/integration test) if any, please explain if no tests are needed.

Ⅳ. Describe how to verify it

Ⅴ. Special notes for reviews

### CODE REVIEW

**Review by JiahuiZhao11 — COMMENTED** [review]
LGTM

File: pkg/apiserver/cubeapi/resourcemanage/resources/service/service.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+// get external ips
+func (s *Service) addExtendInfo(serviceList corev1.ServiceList) resources.K8sJsonArr {
+	resultList := make(resources.K8sJsonArr, 0)
+
+	for _, service := range serviceList.Items {
+		ips := make([]string, 0)
+
+		switch service.Spec.Type {
+		case "NodePort":
```
Comment (weilaaa) [inline pkg/apiserver/cubeapi/resourcemanage/resources/service/service.go:87]: use constants replaced: pkg/apis/core/types.go

Comment (fusida) [reply pkg/apiserver/cubeapi/resourcemanage/resources/service/service.go]: done

**Review by weilaaa — APPROVED** [review]
LGTM

**Pull request merged**